src/Components/NewAdminForm/style.module.css

```diff
@@ -0,0 +1,118 @@
+@import "../../colors.module.css";
+@import "../../fonts.module.css";
+
+
+.container {
+  min-width: 0px;
+  flex-wrap: wrap;
+  max-width: 100%
+}
+
+.newAdminForm {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  padding-right: 10%;
+  padding-left: 10%;
+}
+
+.fieldContainer {
+  width: 90%;
+  align-items: center;
+  justify-content: space-evenly;
+  display: flex;
+  flex-direction: row;
+  padding: 2%
+}
+
+.buttonContainer {
+  display: flex;
+  flex-direction: row;
+  justify-content: end;
+  gap: 5%;
+  padding-right: 11%;
+}
+
+
+#buttonConfirm {
+  width: 150px;
```

Comment:
not the biggest deal, but try to use `rem` or percentages in most places instead of hard pixels